packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -858,25 +863,28 @@ export default defineComponent({
       const filesToSkip = resolvedFileConflicts
         .filter((e) => e.strategy === ResolveStrategy.SKIP)
         .map((e) => e.name)
-      const filesToKeepBoth = resolvedFileConflicts
-        .filter((e) => e.strategy === ResolveStrategy.KEEP_BOTH)
+      const foldersToSkip = resolvedFolderConflicts
+        .filter((e) => e.strategy === ResolveStrategy.SKIP)
         .map((e) => e.name)
 
-      for (const fileName of filesToKeepBoth) {
-        const file = files.find((e) => e.name === fileName && !e.meta.relativeFolder)
-        const extension = extractExtensionFromFile({ name: fileName } as Resource)
-        file.name = resolveFileNameDuplicate(fileName, extension, this.files)
-      }
-
       files = files.filter((e) => !filesToSkip.includes(e.name))
+      files = files.filter(
+        (file) =>
+          !foldersToSkip.some((folderName) => file.meta.relativeFolder.split('/')[1] === folderName)
```

Comment:
Please include the condition in line 870. Doesn't make much sense iterating twice for filtering, when you can do all the filtering in one iteration.

Filter should look like this:
```js
files = files.filter(
        (file) =>
          !filesToSkip.includes(file.name) &&
          !foldersToSkip.some((folderName) => file.meta.relativeFolder.split('/')[1] === folderName)
      )
```